server.go

```diff
@@ -128,6 +128,10 @@ func (s *Server) Broadcast(msg string, except *Client) {
 			}
 			client.Send(personalMsg)
 		} else {
+			if client.quietMode && strings.HasPrefix(msg, systemMessageFormat) {
```

Comment:
Ah, I didn't think about the first point - I wasn't aware users could have the same color.

How does 3 regexps sound to filter our renaming/joining/leaving?

Will update the else if. :smile: